tests/test_grains.py

```diff
@@ -14,19 +13,6 @@
 TOLERANCE = {"atol": 1e-07, "rtol": 1e-07}
 
 
-def test_gaussian_filter(small_array_grains: Grains, grains_config: dict) -> None:
```

Comment:
I think this test should be moved to `test_filters_minicircle.py` so that we retain a unittest of the `Filters.gaussian_filter()` against `minicircle.spm`.

Reply:
This test has been moved to `test_filters.py` already (lines 178-189). Although, neither this nor the earlier test were agains `minicircle.spm`, only the random array.

Reply:
The more tests the merrier - I noticed there wasn't one so I've added a test here